src/plugins/identity/alias.js

```diff
@@ -64,3 +68,31 @@ export function resolveAlias(
       throw new Error(`Unknown type for alias: ${alias}`);
   }
 }
+
+/**
+ * Attempt to convert a NodeAddressT to an Alias.
+ *
+ * If the provided node address corresponds to a known aliasing scheme, an
+ * alias will be returned. Otherwise, null is returned. Since it's possible
+ * that the node address is a valid user node address provided by a plugin that
+ * didn't update this alias registry, clients should endeavor to accomodate
+ * those addresses rather than erroring.
+ */
+export function toAlias(n: NodeAddressT): Alias | null {
+  const parts = NodeAddress.toParts(n);
+  const terminator = parts[parts.length - 1];
+  let prefix = null;
+  if (NodeAddress.hasPrefix(n, githubUserType.prefix)) {
+    prefix = "github";
+  }
+  if (NodeAddress.hasPrefix(n, discourseUserType.prefix)) {
+    prefix = "discourse";
+  }
+  if (NodeAddress.hasPrefix(n, identityType.prefix)) {
+    prefix = "sourcecred";
+  }
```

Comment:
Nit: these should be mutually exclusive. Maybe use `else if`, early returns, or iife assignments?

Reply:
Actually, we could probably anticipate on the future "register your own resolvers" pattern, while solving this nit at the same time. What do you think about looping through a mapping?

```js
  const prefixes: Map<string, NodeAddressT> = new Map([
    ["github", githubUserType.prefix],
    ["discourse", discourseUserType.prefix],
    ["sourcecred", identityType.prefix],
  ]);

  for(const [prefix, nodePrefix] of prefixes.entries()) {
    if(NodeAddress.hasPrefix(n, nodePrefix)) {
      return `${prefix}/${terminator}`;
    }
  }

  return null;
```

Reply:
Yeah, that looks good to me. I'll do that.